python-sdk/example_dags/example_load_file.py

```diff
@@ -142,7 +141,7 @@
     # [START load_file_example_12]
     aql.load_file(
         input_file=File(
-            "gs://astro-sdk/workspace/sample_pattern",
+            "gs://astro-sdk/workspace/sample_pattern.csv",
```

Comment:
Look like old resource does not exist I'm getting 404 in CI for old path

Reply:
hmm, it should have worked no? Or else the tests in main would have failed?

Reply:
actually I was not able to find the file too, how is this working? or do we not run this tests on main?

Reply:
created https://github.com/astronomer/astro-sdk/issues/1163 to address this issue